plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java

```diff
@@ -108,4 +120,9 @@ public static Optional<Boolean> getChangeDataFeedEnabled(Map<String, Object> tab
     {
         return Optional.ofNullable((Boolean) tableProperties.get(CHANGE_DATA_FEED_ENABLED_PROPERTY));
     }
+
+    public static ColumnMappingMode getColumnMappingMode(Map<String, Object> tableProperties)
+    {
+        return ColumnMappingMode.valueOf(tableProperties.get(COLUMN_MAPPING_MODE_PROPERTY).toString().toUpperCase(ENGLISH));
```

Comment:
```
return (ColumnMappingMode) tableProperties.get(COLUMN_MAPPING_MODE_PROPERTY);
```

Reply:
I kept the current code because I changed to `stringProperty`.